src/components/Tank/Tank.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { getBgPosition, getTankPositions } from './Tank.helpers';
+import { TankProps } from './Tank.model';
+import styles from './Tank.module.scss';
+
+const propsDefault: Readonly<Required<TankProps>> = {
+  color: 'yellow',
+  position: 'up',
+  type: '0'
+};
+
+// eslint-disable-next-line complexity
+const Tank: FC<TankProps> = (
+  props: Readonly<TankProps>
+): JSX.Element => {
```

Comment:
- [ ] I think you could make this to be in a single line as our line length is 128. And the return type could be ReactElement instead of JSX.Element.